two_factor/views/core.py

```diff
@@ -213,7 +213,10 @@ class SetupView(IdempotentSessionWizardView):
         ('validation', DeviceValidationForm),
         ('yubikey', YubiKeyDeviceForm),
     )
+
+    show_welcome = not hasattr(settings, 'TWO_FACTOR_SKIP_WELCOME') or not settings.TWO_FACTOR_SKIP_WELCOME
```

Comment:
1. because a definition of `None` in the settings is not the same as `False`.
2. because it will still be executed at import time, not when it should be evaluated.